public/button.js

```diff
@@ -40,11 +40,13 @@ const updateTimer = () => {
     const delta = Math.floor(new Date().getTime() / 1000) - start;
     const sec = delta % 60;
     const min = Math.floor((delta / 60) % 60);
-    const hour = Math.floor(delta / 3600);
+    const hour = Math.floor((delta / 3600) % 3600);
+    const days = Math.floor(delta/(3600 * 24))
```

Comment:
Spaces around / operator, and semi colon
```suggestion
    const days = Math.floor(delta / (3600 * 24));
```